addons/point_of_sale/views/pos_assets_common.xml

```diff
@@ -97,7 +97,6 @@
 </template>
 
 <template id="point_of_sale.pos_assets_backend" name="POS UI Backend Assets" inherit_id="web.assets_backend" primary="True">
-    <xpath expr="//script[contains(@src, '/mail/static/src/js/services/mail_service.js')]" position="replace"></xpath>
```

Comment:
@alexkuhn you removed mail service and mail manager? Is activity handler removed too then?

Reply:
mail manager handled activity_updated by just triggering this event on core.bus.
Just managed that in messaging_notification_handler, so could have just dropped mail service altogether.

Reply:
Ok I just checked _handleNotificationPartner, but replace Component.env by this.env there